proposals/kubeflow-steering-committee-election-proposal.md

```diff
@@ -0,0 +1,125 @@
+
+This dashboard **\<TODO\>** shows only GitHub based contributions and does not capture all the contributions we value. We expect this metric not to capture everyone who should be eligible to vote. If a community member has had significant contributions over the past year but is not captured in the dashboard, they will be able to submit an exception form to the Elections Officer(s) who will then review and determine whether this member should be eligible to vote. All exceptions, and the reasons for them, will be recorded in a log that will be available to Steering and the TOC.
+
+The electoral roll of all eligible voters will be captured at kubeflow/community/steering-elections/$YEAR/voters.md and the voters' guide will be captured at kubeflow/community/steering-elections/\$YEAR/README.md, similar to the kubernetes election process and identical to the TOC election process.
+
+We are committed to an inclusive process and will adapt future eligibility requirements based on community feedback.
+
+## Candidate Eligibility ##
+Community members must be eligible to vote in order to stand for election (this includes voters who qualify for an exception). Candidates may self-nominate or be nominated by another eligible member. There are no term limits for KSC members. Nothing prevents a qualified member from serving on both the TOC and SC simultaneously.  
+
+### Voting Procedure ###
+Elections will be held using [Elekto](https://elekto.dev/), an online voting tool created by CNCF intern Manish Sahani. The details for previous KSC elections can be found in the elections folder **\<TBD\>**.
+
+### Limitations on Company Representation ###
+No more than two seats may be held by employees of the same organization (or conglomerate, in the case of companies owning each other). If the results of an election result in greater than two employees of the same organization, the lowest vote getters from any particular employer will be removed until representation on the committee is down to two.  
+
+If employers change because of job changes, acquisitions, or other events, in a way that would yield more than 2 seats being held by employees of the same organization, sufficient members of the committee must resign until only two employees of the same employer are left. If it is impossible to find sufficient members to resign, all employees of that organization will be removed and new special elections held. In the event of a question of company membership (for example evaluating independence of corporate subsidiaries) a majority of all non-involved Steering Committee members will decide.
+
+### Vacancies ###
+In the event of a resignation or other loss of an elected committee member, the next most preferred candidate from the previous election will be offered the seat.  
```

Comment:
@zijianjoy also made a comment about place a minimum vote requirement on selecting the "next most preferred candidate", which I think is supported by elekto tool in some way (see [ranking](https://elekto.dev/docs/administration/concluding/#publishing-the-results) mentioned in their doc). On the other hand, I'd like to reduce process if possible. I am interested in hearing from others.
@jbottum @theadactyl @andreyvelich @johnugeorge @annajung

Reply:
I agree, we can start with simple rule and improve it during our evaluations.

Reply:
Just to accommodate for the possibility of needing to run a special election instead of filling the empty seat with the previous candidate, we can modify the doc to say "... the next most preferred or conduct a special election to fill the empty seat"

Reply:
@annajung I'd like to be more prescriptive here.

It looks like there is a mix of opinions here. I think that elections can be very expensive in terms of time and effort. Not only we need candidates, but also election officers to volunteer their time and effort to a multi-week/month process. We can probably get an idea of the level of effort needed from our 1st election. For now I am going to keep the process as-in, and we can re-evaluate after our first election.